flang/test/Semantics/kinds04_q10.f90

```diff
@@ -7,7 +7,7 @@
 !
 ! This test is for x86_64, where exponent-letter 'q' is for
 ! 10-byte extended precision
-! UNSUPPORTED: system-windows
+! UNSUPPORTED: system-windows, target=powerpc{{.*}}
```

Comment:
This is rather a question. 
Should the test case specify REQUIRED platform instead of UNSUPPORTED?

Reply:
it seems that previously it was excluded using unsupported not sure if we should exclude it from other platforms by using required